### ISSUE TITLE
Configuring role ARN in ~/.aws/config

### ISSUE THREAD

**Issue by rtimush** [aws-azure-login/aws-azure-login#6]
Thank you for the tool, it work great. One minor annoyance I found when working with multiple roles, is that you always have to select the right one. I would prefer to have a role preconfigured by profile, for example:
```
[profile Administrator]
azure_tenant_id=...
azure_app_id_uri=...
azure_default_username=...
azure_role_arn=arn:aws:iam::123456789012:role/Administrator

[profile Developer]
azure_tenant_id=...
azure_app_id_uri=...
azure_default_username=...
azure_role_arn=arn:aws:iam::123456789012:role/Developer
```

**Comment by dtjohnson** [comment]
Thanks!. Just published v0.5.0 with your PR. I did make a minor tweak to the prompt.

One quick question if you don't mind. How do you configure multiple roles for an app in Azure AD? (I'm interested in doing this myself.)

**Comment by rtimush** [comment]
You need to edit Azure application manifest (better using https://apps.dev.microsoft.com, editing from https://portal.azure.com has some issues). There you can add as many `appRoles` as you want, just be sure to generate an unique id for each role.
